WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -19,6 +19,16 @@ def extract_model_name(key)
     when "devise" then "user"
     when "admin" then "person"
     when "oauth" then "doorkeeper/application"
+    when "contact"
+      # ContactsController uses WebsiteContact or DobCotact (which are much like extended models).
+      # We need to determine which one does the key refer to.
+      if key.include?("website")
+        "website_contact"
+      elsif key.include?("dob")
+        "dob_contact"
+      else
+        throw "Unrecognized contact model. Key: #{key}"
+      end
```

Comment:
@viroulep Note: the `key` could be like `contacts.dob.*` or `contacts.website.*`. In both cases `contact` is inferred as the model name, but it's in this particular case, so we make another check and correct the inferred model.